packages/connector-sendgrid-mail/src/types.ts

```diff
@@ -0,0 +1,119 @@
+import { Nullable } from '@silverhand/essentials';
+import { z } from 'zod';
+
+/**
+ * @doc https://docs.sendgrid.com/api-reference/mail-send/mail-send#body
+ */
+
+export enum ContextType {
+  'TEXT' = 'text/plain',
+  'HTML' = 'text/html',
+}
+
+export type EmailData = {
```

Comment:
maybe just name it `Email`?

Reply:
Used EmailData to align with provider's docs for better understanding.